### DISCUSSION TITLE
Neu hier, Priortät im sunnyportal ging verloren

### DISCUSSION THREAD

**Discussion by rostausk** [camueller/SmartApplianceEnabler#578] [Q&A]
Hallo, mit dem Umstieg auf SHM2.0 wurden unsere blutooth Steckdosen wertlos. Da kommt der SAE mit seiner breiten Anwendbarkeit gerade recht. Danke dafür schonmal!
Mein 1. SAE Gerät ist ein bisher per bluetooth Steckdose automatisierter Geschirrspüler. Ich richtige ihn mit Anlaufstromerkennung über ein shellyplug geschaltet ein.
Er wurde auf Anhieb als neues Gerät im sunnyportal wahrgenommen. In den Einstellmöglichkeiten des Verbrauchers fehlen jedoch die gern genutzte Prioriät sowie der Anteil an PV-Überschuss /Abregelenergie. Was mach ich falsch? Ich bin gespannt...

**Comment by camueller** [comment]
Bei Geräten, die über SEMP (also den SAE) gesteuert werden, kann die Prioriät nicht konfiguriert werden. Die Priorität im Portal bezieht sich nur auf KANN-Verbraucher, zu denen der Geschirrspüler sicher nicht gehört. Insofern ist das kein Problem.

Den Anteil an PV-Überschuss kann man allerdings sehr wohl konfigurieren unter 
`Verbraucherübersicht  und -planung -> [Verbraucher wählen] -> Konfiguration -> Bearbeiten`

**Reply by rostausk** [reply]
Vielen Dank für deine prompte Antwort!
Wenn ich es recht erinnere, stand die Option "Anteil PV-Überschuss" ohnehin nur für KANN Verbraucher zur Verfügung. Würde ich für den Betrieb von MUSS Verbrauchern PV-Überschuss welchen Anteils auch immer voraussetzen, wäre es keine MUSS Verbraucher mehr, oder?
Im übrigen waren bei uns bisher alle Großverbraucher KANN Verbraucher, weil keinen Strom kaufen unser liebster Sport ist.
Im Falle des Geschirrspülers ist das Bearbeiten Fenster im Portal heute leider sehr begrenzt:
![grafik](https://github.com/camueller/SmartApplianceEnabler/assets/153227232/b1328797-b173-4839-9689-bbfa6e05b6f9)
Ganz anders der ebenfalls direkt angebundene Heizstab:
![grafik](https://github.com/camueller/SmartApplianceEnabler/assets/153227232/440a0fbe-cd83-46b5-831a-7e3aebf24207)
So hätt ichs gerne auch für die SAE Geräte.
Beste Grüße

**Comment by camueller** [comment]
> Im Falle des Geschirrspülers ist das Bearbeiten Fenster im Portal heute leider sehr begrenzt:

Bist Du sicher, dass Dein Geschirrspüler über den SAE verwaltet wird? Bei mir sehen die Eigenschaften im Sunny Portal anders aus:
![Screenshot_20231208_151400](https://github.com/camueller/SmartApplianceEnabler/assets/4489628/007edfe1-192a-4020-92c4-5a4b7b1c9969)

Warum hast Du die Begrenzung der Wirkleistungseinspeisung noch aktiv? Diese Begrenzung braucht man nicht mehr ...

**Reply by rostausk** [reply]
Bevor ich ihn im SAE einrichtete, gab es ihn im Portal nicht und alle seine Daten im Portal sind die, die ich im SAE definierte und er steckt in einem shellyplug, die der SHM gar nicht unterstützt. Ich hab daher keine Idee, wie er sich ohne SAE dorthin geschmuggelt haben könnte.
Das Bild deines Geschirrspülers macht mir Hoffnung, dass ich das auch noch hinkriege.
Hast du mal versucht, den Prioschieber zu verstellen? Der sieht immer deaktiv aus, auch wenn er verstellbar ist...
Die Einspeisung ist noch begrenzt, weil das 2014 die Voraussetzung für die Förderung in Form der für 20 Jahre garantierten Einspeisevergütung und den Investitionszuschuss für den Speicher war. Hast du eine Quelle auf die ich mich berufen kann, wenn ich die Begrenzung rausnehme? De facto stört sie gar nicht, weil der Heizstab das Abregeln verhindert, indem er die abzuregelnde Energie in den Pufferspeicher heizt. Von April bis Oktober nutzen wir keine andere Wärmequelle.

**Comment by camueller** [comment]
> Hast du mal versucht, den Prioschieber zu verstellen?

Geht nicht, er ist deaktiviert. Ist aber nicht schlimm, wie ich schon schrieb: Die Prio ist nur relevant bei mehreren Kann-Verbrauchern. Wenn ich Überschüsse habe, gehen sie meist ins E-Auto. Das klappt perfekt im Zusammenspiel SAE und Sunny Home Manager.

**Reply by rostausk** [reply]
Verstehe
Zur Vermeidung von Missverständnissen:
1. wird dein Geschirrspüler vom SAE verwaltet? Ich schaute gerade noch einmal nach: das Portal bietet "optionaler Energiebedarf" nur für KANN Verbraucher an, was zu meiner Logik passt, dass MUSS Verbraucher im Zeitfenster überschussunabhängig bis zu ihrer min Laufzeit laufen MÜSSEN.
2. Ist der Geschirrspüler dein einziger KANN Verbraucher?
3. Zum Priorisieren muss SHM die Geräte selbst verwalten? Das wäre schade, denn das ist eine mit SHM1.0 und den SMA bluetoothdosen von uns viel genutzte Option. Dann müssten wir doch in SHM2.0 kompatible Schalter investieren.
4. Welche Einstellung sollte ich für meinen Geschirrspüler im SAE anpassen, damit er im Portal wie deiner konfigurierbar wird ?]
![Screenshot_20231208-184722](https://github.com/camueller/SmartApplianceEnabler/assets/153227232/80810063-0832-4c45-9c3a-4189619ebd88)
![Screenshot_20231208-184617](https://github.com/camueller/SmartApplianceEnabler/assets/153227232/1d214ebe-9810-496b-b0ce-a573b2576eef)

**Comment by camueller** [comment]
Die Begriffe KANN und MUSS gibt es bei SEMP-Geräten (also solche, die via SAE eingebundenen sind) nicht. Es gibt Bedarfe (hier: Laufzeitbedarf), die ganz oder teilweise über Optionale Energie (also nur bei Überschuss) bestritten werden können. Diese Bedarfe bzw. Anforderungen werde im SAE durch Zeitpläne abgebildet. Ohne Zeitplan kein Bedarf und deshalb auch kein Einschalten durch den Sunny Home Manager. Du hast leider keinen Zeitplan für Deinen Geschirrspüler gepostet. Bei mir sieht er wochentags wie folgt aus: 
![Screenshot_20231208_191240](https://github.com/camueller/SmartApplianceEnabler/assets/4489628/8d153961-17f3-4c18-b4ac-a2c7e019388c)

Ein paar Anmerkungen zu den bei Dir konfigurierten Geräteeigenschaften:
- Du solltest **keine** Werte für Einschalt-/Auschaltdauern konfigurieren
- Unterbrechung erlaubt macht für programmbasierte Geräte (Waschmaschine, Geschirrspüler, ...) keinen Sinn, sondern sollte nur für zustandslose Geräte (z.B. Filterpumpe vom Swimming Pool) verwendet werden

> Ist der Geschirrspüler dein einziger KANN Verbraucher?

Zum Begriff KANN-Verbraucher hatte ich oben schon etwas geschrieben. Der Geschirrspüler ist keiner, weil er innerhalb eines Zeitfensters laufen muss.

> Zum Priorisieren muss SHM die Geräte selbst verwalten?

Wozu machst Du Dir über das Priorisieren Gedanken? Der Sunny Home Manager ist das Energy Management System (nicht Du ;-) ). Du stellst Anforderungen (Zeitfenster, Laufzeit, max. Leistungsaufnahme, ...) und der Sunny Home Manager kümmert sich darum, dass diese bestmöglich umgesetzt werden.

> Welche Einstellung sollte ich für meinen Geschirrspüler im SAE anpassen, damit er im Portal wie deiner konfigurierbar wird ?

Probier es erstmal mit den genannten Anpassungen.

**Reply by rostausk** [reply]
Danke für deine wertvollen höchst wirksamen Tipps!
Bisher hatte nur der SEMP angebundene Heizstab die Energieoption und kann priorisiert werden. Nun kann auch der SAE (SEMP) Geschirrspüler beides ;-)
![Screenshot_20231209-121853](https://github.com/camueller/SmartApplianceEnabler/assets/153227232/4ff9de0d-b222-4735-94f3-7702a5e385f5)
Wozu Prio?
Wie soll der SHM für die KANNS sonst wissen, dass zuerst das Auto Überschuss kriegen soll, welches zuerst fahren soll, während für das an der 2. wallbox übrigbleibende Krümel genügen? Welchen von 2 identischen Verbrauchern würde er einschalten, wenn der Überschuss nur für einen genügt von dem mir aber 1 wichtiger ist? Prio auf Speicherladen schafft Autarkie über Nacht statt tagsüber Spielzeug zu betreiben. SMA hat verstanden, dass des Nutzers Wunsch sein Himmelreich ist und der SHM ihn mit Prio leicht dorthin führen kann.

**Comment by camueller** [comment]
Klingt ja so, als ob jetzt alles passt :-)

Ich habe nur einen Verbraucher für Überschuss-Energie - meine Wallbox. Daher musste ich mich bisher nicht mit der Priorität beschäftigen.
